docs/Changelog.md

```diff
@@ -53,6 +53,7 @@
 * [#5641](https://github.com/TouK/nussknacker/pull/5641) Fix: fetching/parsing batch periodic json only when needed (stop parsing during status check)
 * [#5656](https://github.com/TouK/nussknacker/pull/5656) Added: Decision Table component - detailed validation
 * [#5657](https://github.com/TouK/nussknacker/pull/5657) Improved heuristic for eventhub to Azure's schema name mapping.
+* [#5754](https://github.com/TouK/nussknacker/pull/5754) Fix test from file, revert 0d9b600
```

Comment:
I propose:
> Fix for not working testing mechanism with Avro format 

Because:
- It is not only for data from file but also ad-hoc
- For jsons it worked

Reply:
It's not only Avro, `InputMetaToJson` also doesn't work properly.  Maybe: Fix for not working encoding mechanism in the tests?